[FW][I18N] account_payment_group: customer receipt template translation

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/account-payment#475

cav-adhoc committed:
[I18N] account_payment_group: customer receipt template translation

X-original-commit: 1144e70d8114c159f342d3dfa6f51010b735bd81

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/476).

fw-bot-adhoc commented:
@cav-adhoc @rov-adhoc cherrypicking of pull request ingadhoc/account-payment#475 failed.

stdout:
```
CONFLICT (modify/delete): account_payment_group/i18n/es.po deleted in HEAD and modified in 16d7ae00 ([I18N] account_payment_group: customer receipt template translation). Version 16d7ae00 ([I18N] account_payment_group: customer receipt template translation) of account_payment_group/i18n/es.po left in tree.
CONFLICT (modify/delete): account_payment_group/__manifest__.py deleted in HEAD and modified in 16d7ae00 ([I18N] account_payment_group: customer receipt template translation). Version 16d7ae00 ([I18N] account_payment_group: customer receipt template translation) of account_payment_group/__manifest__.py left in tree.

```

stderr:
```
16:08:28.726033 git.c:444               trace: built-in: git cherry-pick 16d7ae0093c88ad93b5830dedaed966d5a565a0a
error: could not apply 16d7ae00... [I18N] account_payment_group: customer receipt template translation
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

cav-adhoc commented:
El módulo payment_group está depreciado en v17
